packages/website/src/pages/index.tsx

```diff
@@ -75,16 +85,22 @@ const IndexPage = props => (
     <Section>
       <Flex flexWrap="wrap">
         <Box width={[1, 1 / 2, 2 / 3]} paddingRight={[0, 'medium', 'xxLarge']}>
-          <H2>Seriously fast testing</H2>
+          <H2>Tests that are quick to write, run, and maintain</H2>
           <Text>
-            When you're trying to test from the perspective of your user, there's a lot of information coming in.
-            All the clicks, streaming updates from the server, and the hundreds (or thousands) of background
-            processes create a highly asynchronous environment. As a result, UI testing has historically been
-            time-intensive in both setup and run time.
+            Tests are tools to help us see what we don’t know; they alert us to what is broken when we thought
+            something was unaffected. The best tests swiftly and precisely diagnose the issue so teams can iterate
```

Comment:
`The best tests swiftly and precisely diagnose its issues so teams can iterate`

Reply:
I think for this one, it's safe to use plural because it doesn't necessarily imply there will always be multiple (because of bad code) but just a grammatical issue.